### DISCUSSION TITLE
re-read custom CleanerML

### DISCUSSION THREAD

**Discussion by FrankBKK** [bleachbit/bleachbit#1690] [Q&A]
I am using Bleachbit on Windows - Is there a way to re-read a custom cleaner.xml in the Bleachbit GUI or do I have to restart Bleachbit on every change I make to the  xml file ?

**Comment by az0** [comment]
Open BleachBit preferences and toggle the auto-hide option. It will refresh the list, even if you click it twice, so that it is back to the original setting. Does this help?
